tests/src/test/java/io/streamnative/pulsar/handlers/amqp/AmqpTokenAuthenticationTestBase.java

```diff
@@ -44,10 +45,15 @@ public class AmqpTokenAuthenticationTestBase extends AmqpProtocolHandlerTestBase
     @Override
     public void setup() throws Exception {
         conf.setAuthenticationEnabled(true);
-        conf.setAuthenticationProviders(Sets.newHashSet(AuthenticationProviderToken.class.getName()));
-
+        conf.setAuthenticationProviders(Sets.newHashSet(
+                AuthenticationProviderToken.class.getName(),
+                AuthenticationProviderBasic.class.getName()
+        ));
         Properties properties = new Properties();
         properties.setProperty("tokenSecretKey", AuthTokenUtils.encodeKeyBase64(secretKey));
+        String basicAuthConf =
```

Comment:
Could you test multi users? Maybe we could provide a util to generate this base64 string.

Reply:
Done.